ansible/roles/digidecs/templates/digidecs.service.j2

```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=Run the digidecs backend
+OnFailure=failure-notificator@%n.service
```

Comment:
Can't check on mobile, but did you add the matching healthcheck on healthchecks.io?

Reply:
Not sure how to do that tbh. Couldn't find one for fallacious-rooster, on which this service is based

Reply:
Appearently we have to ways of monitoring our services. This `OnFailure` is used for every systemd service, whilst only some of them use Healthchecks.io. I will have to ask when to use healthchecks.io, but this comment is resolved.